tests/persistAsync.test.tsx

```diff
@@ -5,8 +5,6 @@ import { persist } from '../src/middleware'
 
 describe('persist middleware with async configuration', () => {
   it('can rehydrate state', async () => {
-    let postRehydrationCallbackCallCount = 0
```

Comment:
Those counts are no longer necessary as jest will fail the test if the callback wasn't called. Nice.